packages/frontend/src/components/announcement/logos/AnnouncementLogos.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+
+export function L2BeatLogo() {
```

Comment:
We already have our logo in the codebase

Reply:
https://github.com/l2beat/l2beat/pull/1692/commits/4640c83ac2cfcabff5aa32479a69f25ce45c2447